Library/Homebrew/utils/pypi.rb

```diff
@@ -91,7 +91,7 @@ def to_s
 
     sig { params(other: Package).returns(T::Boolean) }
     def same_package?(other)
-      @name.tr("_", "-") == other.name.tr("_", "-")
+      @name.downcase.tr("_", "-") == other.name.downcase.tr("_", "-")
```

Comment:
```suggestion
      @name.tr("_", "-").casecmp(other.name.tr("_", "-")).zero?
```

Avoids creating new string objects.

Reply:
Thanks!